sdk/identity/Azure.Identity/src/AzureAuthorityHosts.cs

```diff
@@ -27,6 +27,7 @@ public static class AzureAuthorityHosts
         /// <summary>
         /// The host of the Microsoft Entra authority for tenants in the Azure German Cloud.
         /// </summary>
+        [Obsolete("Microsoft Cloud Germany was closed on October 29th, 2021.")]
```

Comment:
Let's add this also. 

```suggestion
        [Obsolete("Microsoft Cloud Germany was closed on October 29th, 2021.")]
        [EditorBrowsable(EditorBrowsableState.Never)]
```

Reply:
@christothes I'd rather not exclude it from IntelliSense, if you don't mind - my setup explicitly required me to configure the authority URL & I was more than happy to find it (after a lot of trial & error, that is 😅).

Reply:
OK, that is fair.